docs/pull_request_template.md

```diff
@@ -1,14 +1,56 @@
 # Zero To Mastery - Webblocks 2022 📦📦
 
-👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈
-
-## PR contains:
+ ## PR contains:
 - INFO ABOUT YOUR PULL REQUEST GOES HERE (Please be as descriptive as possible) 🤜
-
+<!---
 ## Breaking changes
 - IF ANYTHING YOU'RE COMMITTING WOULD BREAK SOMETHING, INCLUDE HERE WHAT WOULD BREAK
 - IF YOU HAVE NO BREAKING CHANGES, ENTER 'None'
 
 ## Screenshots
 - IF YOU HAVE ANY SCREENSHOTS, INCLUDE THEM HERE.
-- IF YOU HAVE NO SCREENSHOTS, ENTER 'None'
\ No newline at end of file
+- IF YOU HAVE NO SCREENSHOTS, ENTER 'None'
+
+## Related Issue
+- Mention issue for which this pull request is a part of (please mention issue number # )
+
+## Proposed Changes
+- Change 1
+- Chnage 2
+
+## Additional Info
+- Any additional information or context
+ -->
+## Checklist
+
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] ✅ My code follows the code style of this project.
+- [ ] 📝 My change requires a change to the documentation.
+- [ ] 🎀 I have updated the documentation accordingly.
+- [ ] 🌟 ed the repo
```

Comment:
Contributors shouldn't be forced to star the repo.

Reply:
I have removed this checklist option .